app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  
+  def current_user
+    @user ||= User.find_by(id: session[:user_id])
+  end
```

Comment:
No need to define this method again. In this application controller you can declare this method as a helper method